### PR TITLE
including links in nodegirls

### PR DESCRIPTION
**Pull request by hdrdavies**
-Changed Dec issue file as it won't be a full issue
-touched up some of the titles 
-embedded links

### CODE REVIEW

File: 2015-Nov/draft.md [hunk 1]
```diff
@@ -7,57 +7,42 @@ We recently decided to start publishing a monthly newsletter to keep the communi
...
-
-#### Cohort Project of the month
-
---possible addition here--
-
-#### Alumni project highlights
+FAC5 graduates, [Anita](https://twitter.com/a_nitacz), [Anni](https://twitter.com/msmichellegar), [Claire](https://twitter.com/nofootnotes), [Michelle](https://twitter.com/msmichellegar), [Mina](https://twitter.com/minaorangina), and [Nikhila](https://twitter.com/nikhilaravi), founded [Node Girls](https://twitter.com/nodegirls_ldn) and, together with [Ines](twitter.com/iteles), [Natalia](github.com/nataliaLKB) (FAC3) and others, organised the inaugural one-day bootcamp in Node.js for women at Stack Exchange in London on Saturday 7th November. The event was a great success and [was well received](https://twitter.com/elinnett/status/663469821976305665). We are expecting more great Node Girls events in the New Year.
```
Comment (iteles) [inline 2015-Nov/draft.md]: Two notes:
+ Anni's twitter is https://twitter.com/anni_vaananen
+ Again I would argue that at least Nat should be listed as a founding member.

File: 2015-Nov/draft.md [hunk 2]
```diff
@@ -7,57 +7,42 @@ We recently decided to start publishing a monthly newsletter to keep the communi
...
 
-https://www.instagram.com/modeforme/  
-https://twitter.com/modeforme  
-http://www.modeforme.com/
+Well done Claire and Michelle! Michelle is now working with WonderLuk and Claire is at 27partners. We wish them success with Mode For Me.
 
 #### FAC Goodbyes
 
-We were very sad to lose Benjamin Lees to Arcadia group in late October. We wish him all the best in his role and hope to see him very soon. --Benji's bit--.
+We were very sad to lose Benjamin Lees (FAC3) to Arcadia group in late October. We wish him all the best in his role and hope to see him very soon. --Benji's bit--.
```
Comment (iteles) [inline 2015-Nov/draft.md]: And Claire!

File: 2015-Nov/draft.md (hunk 1 above)
Comment (msmichellegar) [inline 2015-Nov/draft.md]: Can we just say "women of FAC" put together the first Node Girls?

**Comment by hdrdavies** [comment]
Claire didn't want a goodbye!

**Comment by hdrdavies** [comment]
@iteles what do you make of "women of FAC" as @msmichellegar put it?

File: 2015-Nov/draft.md (hunk 1 above)
Comment (iteles) [inline 2015-Nov/draft.md]: @msmichellegar I like that better! Or Ladies of FAC?

**Comment by iteles** [comment]
@hdrdavies YES. Much better than all this back and forth :+1: 
Ladies or women work for me.

**Comment by hdrdavies** [comment]
@iteles @msmichellegar changed - if you're happy, please merge and we can open it up to more eyes to have a look over

File: 2015-Nov/draft.md [hunk 3]
```diff
@@ -7,57 +7,46 @@ We recently decided to start publishing a monthly newsletter to keep the communi
 #### FAC Developments
 
 ##### A New Home
-We are currently located in Bethnal Green, at 14 Palmers Road, in a space which is very dear to us. However, we are looking for another nest as we are growing very fast with the current cohort soon ending! If you are contact with anyone who could provide a place for running free education we would be hugely indebted to you.
+We are currently located in Bethnal Green, at 14 Palmers Road, in a space which is very dear to us. However, we are looking for another nest as we are growing *very* fast with the current cohort soon ending! If you are contact with anyone who could provide a place for running free education we would be hugely indebted to you.
```
Comment (msmichellegar) [inline 2015-Nov/draft.md]: Grammar error: "if you are *in* contact with anyone".

**Comment by msmichellegar** [comment]
All good except for the grammar error! Do you have time to fix it quickly?

**Comment by hdrdavies** [comment]
@msmichellegar can do it later today. It's Ronan workshop o'clock right now

**Comment by nofootnotes** [comment]
Fixed the grammar for ya @hdrdavies

**Comment by hdrdavies** [comment]
@nofootnotes what a babe! thank you

**Comment by hdrdavies** [comment]
Thanks all for input. Really appreciate it :+1:

**Pull request merged**